app/src/sharedTest/java/org/oppia/android/app/options/ProfileEditActivityTest.kt

```diff
@@ -0,0 +1,179 @@
+package org.oppia.android.app.options
```

Comment:
@DeonWaju I think there is no need of commiting this file.
I'm right @rt4914 ?

Reply:
Ok, Thought @rt4914 said write tests like Please add test similar to [](https://github.com/oppia/oppia-android/pull/3227/files) in ProfileEditActivity.

Reply:
Okay @DeonWaju Thanks for clearing. Yes this file we need then.